docs/azure_arc_jumpstart/azure_arc_k8s/aks_stack_hci/aks_hci_powershell/_index.md

```diff
@@ -0,0 +1,139 @@
+---
+type: docs
+title: "AKS on Azure Stack HCI PowerShell"
+linkTitle: "AKS on Azure Stack HCI PowerShell"
+weight: 1
+description: >
+---
+
+## Deploy AKS cluster on Azure Stack HCI and connect it to Azure Arc using PoweSh
+
+The following README will guide you on how to use the provided PowerShell script to deploy an [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/intro-kubernetes) cluster on [Azure Stack HCI](https://docs.microsoft.com/en-us/azure-stack/hci/overview) and connected it as an Azure Arc cluster resource.
+
+Azure Kubernetes Service on Azure Stack HCI is an implementation of AKS on-premises using hyperconverged infrastructure operating system that is delivered as an Azure service.  
+
+  > **Note: Azure Kubernetes Service is now in preview on Azure Stack HCI and on Windows Server 2019 Datacenter.**
+
+  > **Note: Currently, Azure Arc enabled Kubernetes is in [public preview](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/).**
+
+This guide assumes you already have an existing 2-4 node Azure Stack HCI cluster and will leverage PowerShell to automate the AKS creation and onboarding process of the cluster to Azure Arc. 
```

Comment:
```suggestion
This guide assumes you already have an existing Azure Stack HCI cluster (with 2 to 4 nodes) and will leverage PowerShell to automate the AKS creation and onboarding process of the cluster to Azure Arc. 
```

Reply:
1. Is 4 the supported maximum number of nodes?
2. IMO, the language here should be more decisive in the sense of why we are assuming this and not providing the instructions on deploying HCI.